app/Resources/views/index.html.twig

```diff
@@ -7,6 +7,40 @@
 
 {% block body %}
 
+  {% if not app.user and not isWebview() %}
+    <div id="welcomeSection">
+      <h3>{{ "programs.welcome.title"|trans({}, "catroweb") }}</h3>
+      <h3>{{ "programs.welcome.subtitle"|trans({}, "catroweb") }}</h3>
+      <div class="row">
+        <div class="col-md-8">
+          <img src="{{ asset('images/help/starters.png') }}" height="200px" width="33%" onclick="self.location.href='{{ url('catrobat_web_starter') }}'" />
```

Comment:
Don't use `width: 33%`. We would like to have it dynamic which means you should use another bootstrap grid inside here. Just start a new row and use `col-md-*` again. (nested rows work)

Don't use onclick listeners. The mouse pointer won't change which is bad usability. Use an anchor that contains the image tag and set the href of the anchor and you are golden.

Instead of the images which get distoreted I would love to see you make "cards" using css which look the same as the images you used. It shouldn't be that hard using bootstrap. It would take a while but would be worth the effort.
If you prefer not to, this is ok for now. 
The 2 other changes are mandatory.

Reply:
The css for bootstrap cards seems not to be included in our project.
I tried to use a link to a bootstrap stylesheet, but the cards didn't looked nice.